web/views.py

```
@@ -6955,3 +6973,448 @@ def delete_post(request, post_id):
+        except Exception as e:
+            logger.exception("Unexpected error refreshing membership for user %s: %s", request.user.email, e)
+
+    # Get invoices if customer ID exists
+    invoices = []
+    if membership.stripe_customer_id:
+        try:
+            setup_stripe()
+            invoice_list = stripe.Invoice.list(customer=membership.stripe_customer_id, limit=10)
+            invoices = invoice_list.data
+        except stripe.error.StripeError as e:
+            logger.exception("Error retrieving invoices for user %s: %s", request.user.email, e)
```

Ruff (0.8.2)

Redundant exception object included in `logging.exception` call

(TRY401)